src/wazuh_modules/wm_syscollector.c

```diff
@@ -27,7 +27,7 @@ static void wm_sys_stop(wm_sys_t *sys);         // Module stopper
 const char *WM_SYS_LOCATION = "syscollector";   // Location field for event sending
 cJSON *wm_sys_dump(const wm_sys_t *sys);
 int wm_sync_message(const char *data);
-pthread_cond_t sys_stop_condition;
+pthread_cond_t sys_stop_condition = PTHREAD_COND_INITIALIZER;
```

Comment:
Just for safety, I'd add `PTHREAD_MUTEX_INITIALIZER` to the mutexs defined in the file.